xaynetwork/xayn_discovery_app
# TB-3945 Integrate RevenueCat with AppsFlyer

### What 🕵️ 🔍

Since the Subscription events in analytics won't be reliable to check on Mixpanel and AppsFlyer, we agreed on integrating RevenueCat with AppsFlyer (this PR), and RevenueCat with Mixpanel ([Mixpanel PR](https://github.com/xaynetwork/xayn_discovery_app/pull/461)). Once the user make a purchase, opt to a trial or cancel subscription, super properties on Mixpanel will be updated, and events will be sent from RevenueCat to Mixpanel/AppsFlyer.

----------

### How to test 🥼 🔬
- Test payment workflow
- Check that AppsFlyer platform has events from RevenueCat
- Test payment workflow on [Mixpanel branch](https://github.com/xaynetwork/xayn_discovery_app/pull/461)
- Check that Mixpanel platform has events from RevenueCat

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-3945)

----------
